lib/redis_rails_cache.rb

```diff
@@ -0,0 +1,40 @@
+# Temporary and will be removed after all cache keys have been moved over to Redis
+class RedisRailsCache
+  DEFAULT_EXPIRATION = 86400.freeze
+
+  class << self
+    def fetch(key_name, opts = {})
+      # Default expire all keys after 24 hours if expiration is not given
```

Comment:
This will be available to us in Rails.cache through the configuration setup but for now we have to implement it manually and I definitely don't want any keys going in without expirations. Been there, made that mistake before 🙃